docs/docsite/rst/user_guide/playbooks_reuse.rst

```diff
@@ -44,6 +44,11 @@ You can select which playbook you want to import at runtime by defining your imp
 
 If you run this playbook with ``ansible-playbook my_playbook -e import_from_extra_var=other_playbook.yml``, Ansible imports both one_playbook.yml and other_playbook.yml.
 
+When to turn a playbook into a role
+===================================
+
+For some use cases, simple playbooks work well. However, starting at a certain level of complexity, roles work better than playbooks. A role lets you store your defaults, handlers, variables, and tasks in separate directories, instead of in a single long document. Roles are easy to share on Ansible Galaxy. For complex use cases, most users find roles easier to read, understand, and maintain than all-in-one playbooks.
+
```

Comment:
What comes to mind as I read this intro - 
- would I ever use a role if I didn't have any reuse needs? or are roles most beneficial when I need to give my spiffy new playbook etc to someone else in my org (or make available to other users via Galaxy)?
- are there any features ONLY available to roles (except Galaxy sharing) ?
- Does the new [roles argspec](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-argument-validation) make roles somehow 'better'? Like it validates inputs and maybe spits out errors sooner and/or clearer than if this was all done with a playbook?